README.md

```diff
@@ -10,13 +10,25 @@
 ## Introduction
 A go framework for Aries
```

Comment:
While you are here, I think "go" should be "Go".

Reply:
@troyronda [Updated](https://github.com/hyperledger/aries-framework-go/compare/77de294918da55729525425ed777814198cf215f..480655f1ce24de110974569fbec272fa8c20411f)